worker.go

```diff
@@ -37,6 +37,8 @@ func startWorkers(fileName string, nbWorkers int, env PreparedEnv) error {
 		return fmt.Errorf("workers %q: %w", fileName, err)
 	}
 
+	metrics.RenameWorker(fileName, absFileName)
```

Comment:
I'm not sure to understand what this does. Can't register then metric for this worker only at this point to not have to rename it?

Reply:
So, I have mixed feelings about this...

Workers don't have identities, and the metric stuff needs an identity for the metric. Right now, it is just using the file name. I've already started working on another PR that gives a proper identity (`appWorker` in this example):

```
frankenphp {
  worker appWorker ../testdata/index.php 2
}
```

and if no name is passed it uses the defined path (similar to the metrics actually, so replacing non-alphanumerics with `_`). That would remove the need for `rename` here, I think.

Currently, this is needed for two reasons:

1. We register the metric before renaming. This makes the name something like "testdata_index_php" in the above example, which aligns with the configuration and makes intuitive sense.
2. we lose that "name" just after this point in the code and only use the full path from now on in the worker's lifecycle. So, we need to rename it or somehow pass the original name throughout the code. I chose 'rename' to keep things simple, but I agree it looks wonky.

Reply:
I would prefer to avoid "aliases" (in Symfony we now use the same "ID" everywhere because multiplying aliases makes debugging harder).

I agree that we should use the path defined by the user in the config everywhere. Maybe could we store the original name it in an unexported field?

Reply:
I've updated it to remove the need for renaming the workers; we set it once.

As a side note, I also upgraded some software last night that decided to listen on port 9080 and I cannot disable the software, for reasons. 🤬

I could rant about that all day long, but I parameterized the http port the tests run on so that I could run them.